lib/azure/armrest/configuration.rb

```diff
@@ -307,7 +279,7 @@ def fetch_providers
       end
 
       def fetch_token
-        token_url = File.join(authority_url, tenant_id, 'oauth2/token')
+        token_url = File.join(environment.authority_url, tenant_id, 'oauth2', 'token')
 
```

Comment:
@bronaghs Oh, haha, sorry. That's a harmless change, I just thought it looked better.